csvqb/csvqb/models/cube/csvqb/components/attribute.py

```diff
@@ -31,10 +30,27 @@ def validate_data(self, data: PandasDataTypes) -> List[ValidationError]:
 
 
 @dataclass
-class NewQbAttribute(QbAttribute, UriIdentifiable):
+class NewQbAttributeValue(UriIdentifiable):
     label: str
     description: Optional[str] = field(default=None, repr=False)
-    code_list: Optional[QbCodeList] = field(default=None, repr=False)
+    uri_safe_identifier_override: Optional[str] = field(default=None, repr=False)
+    source_uri: Optional[str] = field(default=None, repr=False)
+    parent_attribute_uri: Optional[str] = field(default=None, repr=False)
```

Comment:
I think this should be `parent_attribute_value_uri`. As it is, `parent_attribute_uri` might confuse people and result in them saying that this attribute value is a child of an attribute - which they shouldn't do.